lib/client/response.go

```diff
@@ -125,6 +125,7 @@ type CongressVotingResult struct {
 		No    uint64 `json:"no"`
 		ABS   uint64 `json:"abs"`
 	} `json:"result"`
+	TotalMembership uint64
```

Comment:
@spikeekips I discussed this with @outsideris.
@outsideris thought that to verify membership count from user-side, each membership's hash and membership.URL should be also included like voting result.

Reply:
@soonkuk Sorry, I misunderstood, I will make it again :)